src/main/msp/msp_box.c

```diff
@@ -142,12 +142,13 @@ int serializeBoxNameFn(sbuf_t *dst, const box_t *box)
     const char* name = NULL;
     int len;
 #if defined(USE_CUSTOM_BOX_NAMES)
-    if (name == NULL
-        && box->boxId >= BOXUSER1 && box->boxId <= BOXUSER4) {
+    if (box->boxId >= BOXUSER1 && box->boxId <= BOXUSER4) {
```

Comment:
The idea was to make all versions symmetrical - try another name source as long as `name == NULL`

Reply:
@ledvinap As name is declared and initialized with NULL on line 142 this condition won't be needed here ?

Reply:
Yes, it is always true. It will be useful only when someone added new branch with higher priority before this one.